### ISSUE TITLE
[🔨 compilation issue]jit_optix_init(): libnvoptix.so.1 could not be loaded 

### ISSUE THREAD

**Issue by Seasandwpy** [mitsuba-renderer/mitsuba3#129]
## Summary

error: 'jit_optix_init(): libnvoptix.so.1 could not be loaded ' when using the cuda_spectral_polarized mode 

## System configuration

I installed the most recent nvidia driver 515.57 and am using CUDA 11.7. The compilation is ok, but when running the polarizer test script, I got the error 'jit_optix_init(): libnvoptix.so.1 could not be loaded -- disabling OptiX backend! Set the DRJIT_LIBOPTIX_PATH environment variable to specify its path.'

However, I didn't find libnvoptix.so.1 library in the system. In theory, it should be installed with the most update driver. I get confused and would like if you got similar problem?

**Comment by dvicini** [comment]
Hi,
Indeed it should all be installed with the driver. You don't have to install CUDA and OptiX manually, all the functionality Mitsuba uses is loaded from the driver. The 515 driver should work, I think I have the same version on my Linux system.

Have you tried using the pre-built Python wheels? (`pip install mitsuba`) Maybe there something went wrong during compilation? 

Is the driver loaded correctly? You can check that using `nvidia-smi`.

**Comment by Seasandwpy** [comment]
I tried the pre-built wheels and it worked fine. But I want to use the cuda_spectral_polarized variant.  I also compiled the "llvm_ad_spectral_polarized" and could run the polarization test scripts. 

I think the problem now is that the libnvoptix.so.1 is not installed to the system somehow. I extracted the 515 driver run file and found the libnvoptix.so.515.57. Then I coped the file manually to /usr/lib/x86_64-linux-gnu/ and renamed it to libnvoptix.so.1.  When I ran:
mi.set_variant('cuda_spectral_polarized')
scene = mi.load_file('scenes/polarizers.xml')
I get another error: Critical Dr.Jit compiler failure: jit_optix_check(): API error 7999 (OPTIX_ERROR_UNKNOWN): "Unknown error" in ../ext/drjit/ext/drjit-core/src/optix_api.cpp:373.
